infoschema/infoschema.go

```diff
@@ -428,17 +428,21 @@ func GetBundle(h InfoSchema, ids []int64) *placement.Bundle {
 		}
 	}
 
+	var newRules []*placement.Rule
+
 	b, ok := h.BundleByName(placement.PDBundleID)
 	if ok {
```

Comment:
if !ok, them newRules is nil.

Reply:
It should not matter though... I will add a comment on this function.

Reply:
Done. Always return a non-nil slice.